### PR TITLE
Updating mero conf

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Updating mero conf

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by sumabai — COMMENTED** [review]
LGTM

File: conf/quincy/baremetal_pipeline/mero_conf.yaml [hunk 1]
```diff
@@ -95,13 +90,11 @@ globals:
             - /dev/sdj
             - /dev/sdk
             - /dev/sdl
-            - /dev/nvme0n1
         - hostname: mero011
```
Comment (pdhiran) [inline conf/quincy/baremetal_pipeline/mero_conf.yaml:93]: Can we have same no of disks across all the OSD nodes? We need to have a homogenous cluster for testing.

This is the current distribution : 
mero001 (node1): No volumes listed.
mero003 (node2): No volumes listed.
mero004 (node3): No volumes listed.
mero005 (node4): 11 volumes listed.
mero007 (node5): 12 volumes listed.
mero009 (node6): 12 volumes listed.
mero011 (node7): 11 volumes listed.
mero013 (node8): 11 volumes listed.
mero014 (node9): 12 volumes listed.
mero015 (node10): No volumes listed.
mero016 (node11): No volumes listed.
mero017 (node12): 12 volumes listed.
mero018 (node13): 12 volumes listed.
mero019 (node14): 12 volumes listed.
mero020 (node15): 12 volumes listed.

Comment (AmarnatReddy) [reply conf/quincy/baremetal_pipeline/mero_conf.yaml:93]: Hi Pawan,
While reimaging, we've noticed that device labels are being assigned randomly. 
To maintain uniformity in our cluster, we calculate the OSD (Object Storage Daemon) devices separately when applying the OSD configuration in a different script.
Can you please verify if the roles within the stretched clusters are evenly balanced?

Regards,
Amarnath

**Comment by pdhiran** [comment]
from the daemon placement POV, the conf file looks good. with the below distribution of hosts among DCs, we can have stretch cluster with same daemons on both sites.

Arbiter node : mero001
DC1 : mero003, mero004, mero005, mero007, mero009, mero011
DC2 : mero017, mero018, mero019, mero020, mero013, mero014

Clients : mero015, mero016

**Review by pdhiran — COMMENTED** [review]
Looks good!

**Review by pdhiran — APPROVED** [review]
Looks good.

**Pull request merged**